libs/interpret/src/lib/MLIDashboard/Controls/WhatIfTab/WhatIfTab.styles.ts

```diff
@@ -128,14 +132,20 @@ export const whatIfTabStyles: () => IProcessedStyleSet<IWhatIfTabStyles> =
       choiceBoxArea: {
         alignItems: "baseline",
         display: "flex",
-        flexDirection: "row"
+        flexDirection: "row",
+        flexFlow: "wrap"
       },
       choiceGroup: {
-        paddingLeft: "30px"
+        // paddingLeft: "30px"
```

Comment:
Is this comment needed? Same as comment below

Reply:
removed both, thanks!